app.js

```diff
@@ -65,3 +65,38 @@ const booksCollection = new BooksCollection();
 
 booksCollection.init();
 booksCollection.displayBooks();
+
+const listButton = document.querySelector('.booklist');
+const addNewButton = document.querySelector('.addition');
+const contactButton = document.querySelector('.contact');
+const awesomeBooksSection = document.querySelector('.show-books');
+const addNewSection = document.querySelector('.add-newbook');
+const contactInfo = document.querySelector('.contact-info');
+listButton.addEventListener('click', () => {
+  awesomeBooksSection.classList.add('show');
+  addNewSection.classList.add('hide');
+  contactInfo.classList.add('hide');
+});
+addNewButton.addEventListener('click', () => {
+  addNewSection.classList.remove('hide');
+  awesomeBooksSection.classList.remove('show');
+  contactInfo.classList.add('hide');
+});
+contactButton.addEventListener('click', () => {
+  contactInfo.classList.remove('hide');
+  awesomeBooksSection.classList.remove('show');
+  addNewSection.classList.add('hide');
+});
+
+function startTime() {
+  /* eslint-disable */
+  const { DateTime } = luxon;
+  /* eslint-enable */
```

Comment:
- [OPTIONAL] It is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉 

- Note: I'm marking this as optional since you're going under the deadline, but be sure to remove it before merging this PR. 👍🏻